en/installation/Linux.md

```diff
@@ -3,21 +3,54 @@
 ## Prerequisites
 A supported environment. See **[Installation Prerequisites](prerequisites.md)**
 
+## Ubuntu versions
+We support Ubuntu 12.10+, however, newer versions of Ubuntu have changed and the steps to install Cocos2d-x and required dependencies are a bit different. This requires some manual effort and this doc will outline what is required.
```

Comment:
Please change to 14.04. 14.04 it is the oldest supported version by Ubuntu.
https://wiki.ubuntu.com/Releases

Reply:
I know, but technically we support 12.10 and you don't know what version of Ubuntu people are are running. I'd hate to say 14.04 and then have a 12.10 user come along and not use out products because they think we don't support them when we do.